src/Concordium/Client/Output.hs

```diff
@@ -949,6 +949,8 @@ showEvent verbose ciM = \case
                             Just $ printf "Account %s was removed from the allow list." (show $ Cbor.chaAccount holder)
                         Right (Cbor.RemoveDenyListEvent holder) ->
                             Just $ printf "Account %s was removed from the deny list." (show $ Cbor.chaAccount holder)
+                        Right (Cbor.Pause pause) ->
```

Comment:
Will the CBOR representation stil have only one variant or will there also be `Cbor.Pause` and `Cbor.Unpause`?